### PR TITLE
Add Module prefix for the module option groups

### PR DESCRIPTION
**Pull request by zonky2**
Hotfix change config labels at content pull down - add "Modules: "

### CODE REVIEW

File: src/EventListener/DataContainer/ModuleDataContainerListener.php [hunk 1]
```diff
@@ -64,14 +65,16 @@ public function getModules(): array
...
         $themeCollection = \ThemeModel::findAll(array('order' => 'name'));
         while ($themeCollection->next()) {
             $modules[$themeCollection->name] = array();
 
             $moduleCollection = \ModuleModel::findBy('pid', $themeCollection->id, array('order' => 'name'));
             if ($moduleCollection) {
                 while ($moduleCollection->next()) {
-                    $modules[$themeCollection->name][$moduleCollection->id] = $moduleCollection->name;
+                    $modules[$GLOBALS['TL_LANG']['merger2']['legend_module']
+                             . $themeCollection->name][$moduleCollection->id] = $moduleCollection->name;
```
Comment (dmolineus) [inline src/EventListener/DataContainer/ModuleDataContainerListener.php]: I don't like the concatenation done inside the key statement. Please make two statements for it to increase readability:

```php
$category           = sprintf($GLOBALS['TL_LANG']['merger2']['legend_module'], $moduleCollection->id);
$modules[$category] = $moduleCollection->name;
```

File: src/Resources/contao/languages/de/tl_module.php [hunk 2]
```diff
@@ -49,3 +50,4 @@
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles']          = 'Artikel von Elternseite bis zur Wurzel erben';
 $GLOBALS['TL_LANG']['merger2']['inherit_articles_fallback']     = 'Artikel der Seite <em>oder</em> Artikel von Elternseite erben';
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles_fallback'] = 'Artikel der Seite <em>oder</em> Artikel von Elternseite bis zur Wurzel erben';
+$GLOBALS['TL_LANG']['merger2']['legend_module']                 = 'Module: ';
```
Comment (dmolineus) [inline src/Resources/contao/languages/de/tl_module.php]: I prefer using a placeholder for the Module label here, see comment above. So the value should be `Module: %s`. To translators may change the order of the label/value.

File: src/Resources/contao/languages/en/tl_module.php [hunk 3]
```diff
@@ -43,3 +44,4 @@
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles']          = 'Article from parent pages';
 $GLOBALS['TL_LANG']['merger2']['inherit_articles_fallback']     = 'Article of this site <em>or</em> from parent page';
 $GLOBALS['TL_LANG']['merger2']['inherit_all_articles_fallback'] = 'Article of this site <em>or</em> from parent pages';
+$GLOBALS['TL_LANG']['merger2']['legend_module']                 = 'Modules: ';
```
Comment (dmolineus) [inline src/Resources/contao/languages/en/tl_module.php]: I prefer using a placeholder for the Module label here, see comment above. So the value should be `Module: %s`. To translators may change the order of the label/value.

File: src/EventListener/DataContainer/ModuleDataContainerListener.php (hunk 1 above)
Comment (zonky2) [reply src/EventListener/DataContainer/ModuleDataContainerListener.php]: changed

File: src/Resources/contao/languages/de/tl_module.php (hunk 2 above)
Comment (zonky2) [reply src/Resources/contao/languages/de/tl_module.php]: changed

File: src/Resources/contao/languages/en/tl_module.php (hunk 3 above)
Comment (zonky2) [reply src/Resources/contao/languages/en/tl_module.php]: changed

**Pull request merged**